.github/workflows/api-waf-apply.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         include:
           - app: bcda
             env: dev
+          - app: dpc
+            env: prod
```

Comment:
I think this could be added to the env list on line 26?

Reply:
Yea, wasn't sure whether BCDA test and sbx environments would be deployed first, so kept it in the include, but it seems like this will go first, so I will make that change